src/api/commands/itemcategory/ItemCategoryGetCommand.ts

```diff
@@ -43,6 +44,23 @@ export class ItemCategoryGetCommand extends BaseCommand implements RpcCommandInt
         }
     }
 
+    public async validate(data: RpcRequest): Promise<RpcRequest> {
+        if (data.params.length < 1) {
+            throw new MessageException('Requires 1 arg');
+        }
+        let tmpCategory;
+        if (typeof data.params[0] === 'number') {
+            tmpCategory = await this.itemCategoryService.findOne(data.params[0]);
+        } else {
+            tmpCategory = await this.itemCategoryService.findOneByKey(data.params[0]);
+        }
+        if (!tmpCategory) {
```

Comment:
If category isnt found, we never get this far as all the findOne methods throw an Exception.
Also, if you want to throw an error about some model not found, use ModelNotFoundException. Usually this is thrown in cases when we need to fetch some other data, which is not found, to get the data that we need.